app/_hub/kong-inc/openid-connect/overview/_index.md

```diff
@@ -1764,6 +1774,69 @@ Nice, as you can see the plugin even added the `X-Consumer-Id` and `X-Consumer-U
 
 > It is possible to make consumer mapping optional and non-authorizing by setting the `config.consumer_optional=true`.
 
+{% if_version gte:3.6.x %}
+## Mutual TLS Client Authentication
```

Comment:
In addition to listing the endpoints , do you think info about flow is helpful? 
Eg  In the auth code flow, if mtls client authentication is configured then Kong will use this method to authenticate to token endpoint and fetch the token

Reply:
good point, added, thanks.